forms-flow-web/src/constants/formConstants.js

```diff
@@ -4,6 +4,7 @@ export const FORM_NAMES = {
     TELEWORK: 'Telework agreement',
     COMPLIANT_1_10: 'Bullying / Misuse of Authority Complaint Form (Article 1.10)',
     INFLUENZA_WORKSITE_REGISTRATION: '2023 Influenza Worksite Registration',
```

Comment:
I'd add a comment to indicate this is a legacy name and can be deleted after a few months (hopefully on the next change to this file).

Reply:
Added comment for the same.